### PR TITLE
only set waypoint when call is shown

### PR DESCRIPTION
**Pull request by LeSiiN**
only set waypoint when call is shown

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -208,20 +208,56 @@ local function removeZones()
...
-            timer = Config.AlertTime * 1000,
+            timer = timer,
         }
     })
 
     addBlip(data, Config.Blips[data.codeName] or data)
+
+    RespondToDispatch:disable(false)
+
+    local startTime = GetGameTimer()  -- Zeitstempel, wenn die Nachricht gesendet wurde
```
Comment (OK1ez) [inline client/main.lua]: remove comments

File: client/main.lua [hunk 2]
```diff
@@ -208,20 +208,56 @@ local function removeZones()
...
+    while timerCheck do
+        Citizen.Wait(1000)  -- Gibt anderen Threads eine Chance, ausgeführt zu werden
```
Comment (OK1ez) [inline client/main.lua]: remove Citizen.

File: client/main.lua (hunk 1 above)
Comment (LeSiiN) [reply client/main.lua]: Done

File: client/main.lua (hunk 2 above)
Comment (LeSiiN) [reply client/main.lua]: Done

**Comment by 5tratz** [comment]
Tested and working fine for me :)

**Pull request merged**